common.mk

```diff
@@ -1,50 +1,38 @@
-ROCKETCHIP_DIR=$(base_dir)/generators/rocket-chip
-TESTCHIP_DIR = $(base_dir)/generators/testchipip
-
-SCALA_VERSION=2.12.4
-SCALA_VERSION_MAJOR=$(basename $(SCALA_VERSION))
-
-SBT ?= java -Xmx2G -Xss8M -XX:MaxPermSize=256M -jar $(ROCKETCHIP_DIR)/sbt-launch.jar ++$(SCALA_VERSION)
-
+#########################################################################################
+# set default shell for make
+#########################################################################################
+SHELL=/bin/bash
+
+#########################################################################################
+# variables to get all *.scala files
+#########################################################################################
 lookup_scala_srcs = $(shell find $(1)/ -iname "*.scala" 2> /dev/null)
```

Comment:
Adding @davidbiancolin comment here that he added to the other PR: "Maybe consider making this chase symbolic links."

Reply:
Added a `-L` to follow symlinks. Seems to work on `firechip` in the `firesim/sim` directory.